openmetadata-spec/src/main/resources/json/schema/entity/services/ingestionPipelines/ingestionPipeline.json

```diff
@@ -11,7 +11,7 @@
       "description": "Type of Pipeline - metadata, usage",
       "type": "string",
       "javaType": "org.openmetadata.schema.entity.services.ingestionPipelines.PipelineType",
-      "enum": ["metadata", "usage", "lineage", "profiler", "TestSuite"]
+      "enum": ["metadata", "usage", "lineage", "profiler", "TestSuite", "DataInsight"]
```

Comment:
we need to come up with a single naming convention here, as we moved from `profiler` to `TestSuite` etc